src/displayapp/screens/Notifications.cpp

```diff
@@ -178,25 +228,38 @@ Notifications::NotificationItem::NotificationItem(const char* title,
                                                   Modes mode,
                                                   Pinetime::Controllers::AlertNotificationService& alertNotificationService,
                                                   Pinetime::Controllers::MotorController& motorController)
-  : mode {mode}, alertNotificationService {alertNotificationService}, motorController {motorController} {
-  lv_obj_t* container1 = lv_cont_create(lv_scr_act(), NULL);
-
-  lv_obj_set_style_local_bg_color(container1, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_MAKE(0x38, 0x38, 0x38));
-  lv_obj_set_style_local_pad_all(container1, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, 10);
-  lv_obj_set_style_local_pad_inner(container1, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, 5);
-  lv_obj_set_style_local_border_width(container1, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, 0);
-
-  lv_obj_set_pos(container1, 0, 50);
-  lv_obj_set_size(container1, LV_HOR_RES, 190);
-
-  lv_cont_set_layout(container1, LV_LAYOUT_COLUMN_LEFT);
-  lv_cont_set_fit(container1, LV_FIT_NONE);
-
-  lv_obj_t* alert_count = lv_label_create(lv_scr_act(), nullptr);
+  : mode {mode}, 
+    alertNotificationService {alertNotificationService}, 
+    motorController {motorController} {
+
+  container = lv_cont_create(lv_scr_act(), nullptr);
+  lv_obj_set_pos(container, 0, 0);
+  lv_obj_set_size(container, LV_HOR_RES, LV_VER_RES);
+  lv_obj_set_style_local_bg_color(container, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_BLACK);
+  lv_obj_set_style_local_pad_all(container, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, 0);
+  lv_obj_set_style_local_pad_inner(container, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, 0);
+  lv_obj_set_style_local_border_width(container, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, 0);
+  
+  page = lv_page_create(container, nullptr);
```

Comment:
Forgot to mention: Previously we had a container just for the message information. Now there is a parent container around the whole notification (category, number, count, and message). This container is used for the animation. I've changed the message contents to be a page to allow for scrolling for longer messages. This may end up being a useability struggle, as there are other SwipeUp / SwipeDown events handled on the parent... 
See what you guys think.